pkg/arvo/app/chat-pull-hook.hoon

```diff
@@ -0,0 +1,260 @@
+/-  *invite-store, hook=chat-pull-hook
+/+  default-agent, verb, dbug, store=chat-store, lib=chat-hooks
+|%
++$  card  card:agent:gall
++$  versioned-state
+  $%  [%0 state-0]
+  ==
++$  state-0
+  $:  tracking=(map path ship)
```

Comment:
Since you’re starting this hook from scratch anyway, could you make this a `(map [=ship =term] ship)` ?

Reply:
Noted. Thanks!